### PR TITLE
Add new steps in the Requirements

### PR DESCRIPTION
**Pull request by aguspesce**
I add the steps to enable the kernel module and create a docker group. 
I also add an example in Manjaro distribution.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,75 @@
...
+
+Create the docker group:
+
+    sudo groupadd docker
+
+Add your user to the docker group:
+
+    sudo usermod -aG docker $USER
+
+Test:
```
Comment (santisoler) [inline README.md]:
```suggestion
Logout and login again. Then test if your user has permissions to run Docker:
```

File: README.md [hunk 2]
```diff
@@ -1,41 +1,75 @@
...
+
+    docker hello-world
+ 
+
+Pull the docker:
```
Comment (santisoler) [inline README.md]: This instructions are not needed. The script pulls the docker container when running it the first time.

Comment (santisoler) [reply README.md]: I would remove these lines.

**Comment by aguspesce** [comment]
I make all suggestions

**Pull request merged**